platform/admin/ui/src/main/webapp/js/controllers/SessionTimeout.controller.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* jshint browser: true */
+/* global define */
+define(['jquery',
+        'underscore',
+        'marionette',
+        'js/wreqr',
+        'js/views/SessionTimeoutModal.view'
+    ], function ($, _, Marionette, wreqr, SessionTimeoutModal) {
+        'use strict';
+        var SessionTimeoutController;
+
+        var _this = null;
+        var idleNoticeDuration = 60000;
+        var idleTimeoutThreshold = 900000;  // Length of inactivity that will trigger user timeout
```

Comment:
❓ Would it be possible to make this configurable?

Reply:
Sure. It's a STIG requirement, though, so it's not something we want modified after the build, right (what happens if an admin changes it to 20min of inactivity)? What's the appropriate place for the configurable value?

Reply:
It's also highly unlikely that a user would ever encounter this during the course of normal usage. In fact, the renewal aspect makes it *less* likely for them to get logged out unexpectedly. Given that, I don't think it needs to be configurable.

Reply:
Different organizations and companies might have different requirements.  It is also possible that they do not want it at all and should be able to disable it.